torchgeo/datasets/eurosat.py

```diff
@@ -0,0 +1,196 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""EuroSat dataset."""
+
+import os
+from functools import lru_cache
+from typing import Callable, Dict, Optional
+
+import numpy as np
+import rasterio
+import torch
+from torch import Tensor
+
+from .geo import VisionDataset
+from .utils import check_integrity, download_and_extract_archive
+
+
+class EuroSat(VisionDataset):
+    r"""EuroSat dataset.
+
+    The `EuroSat <https://github.com/phelber/EuroSAT>`_ dataset is based on Sentinel-2
+    satellite images covering 13 spectral bands and consists of 10 target classes with
+    a total of 27,000 labeled and geo-referenced images.
+
+    Dataset format:
+
+    * rasters are 13-channel GeoTiffs
+    * labels are values in the range [0,9]
+
+    Dataset classes:
+
+    1. Industrial Buildings
```

Comment:
Should this be numbered 0–9?

Reply:
It doesn't match up with the class numbers in the dataset or anything, just a listing of classes